php/pantheon/checks/database.php

```diff
@@ -44,7 +44,7 @@ protected function countRows() {
 		global $wpdb;
 		foreach ( $this->getTables() as $table ) {
 			$this->tables[$table->TABLE_NAME] = $table;
-			if ( preg_match( "#.*_options#s", $table->TABLE_NAME ) ) { 
+			if ( $wpdb->prefix . 'options' ) { 
```

Comment:
`if (_string_)` is no longer a test, but I believe would always return true.
```suggestion
			if ( $table->TABLE_NAME == $wpdb->prefix . 'options' ) { 
```
^ this looks like what I was thinking with my comment, but would invite testing and someone else to check my assumptions here.

Reply:
I think this should do it, but will test to confirm.